qfdmo/models/acteur.py

```diff
@@ -373,6 +379,8 @@ class Meta:
     )
 
     def save(self, *args, **kwargs):
+        # FIXME: if we need to validate the main action in the service propositions
```

Comment:
Je n'ai pas ajouté la validation car je trouvais cela lourd et dur à maintenir pour une fonctionnalité à destination de l'équipe et qui est simple à comprendre

Reply:
Ok pour moi, juste adapter le commentaire 
```suggestion
        # OPTIMIZE: if we need to validate the main action in the service propositions
```

J'ai déjà vu `OPTIMIZE` mais pas sûr sûr que l'usage soit répandu, sinon y a `TODO`